frontend/src/metabase/admin/settings/components/SettingsEmailForm.jsx

```diff
@@ -23,6 +24,10 @@ const SEND_TEST_BUTTON_STATES = {
   success: t`Sent!`,
 };
 
+const isHosted = MetabaseSettings.get("site-url")
```

Comment:
might it be better to imitate the way some of the settings have an accessor?

```js
  adminEmail() {
    return this.get("admin-email");
  }
```

but for is hosted. then when the env var is exposed this can change in one place without the isHosted definiition duplicated in many places?

Reply:
Ah, perfect — I was thinking along these lines but didn't know where to look. I'll amend.